tests/aff4_image_test.cc

```diff
@@ -22,7 +22,7 @@ namespace aff4 {
 
 class AFF4ImageTest: public ::testing::Test {
  protected:
-  std::string filename = "/tmp/aff4_test.zip";
+  std::string filename = "file:///tmp/aff4_test.zip";
```

Comment:
This technically needs to be file_urn and we need to get it from URN::NewURNFromFilename() because AFF4FactoryOpen now always accepts URNs.